core/src/main/java/io/grpc/ServerCall.java

```diff
@@ -50,6 +53,17 @@
  */
 public abstract class ServerCall<RespT> {
   /**
+   * {@link Attributes.Key} for the remote address of server call.
+   */
+  public static final Attributes.Key<SocketAddress> REMOTE_ADDR_KEY =
```

Comment:
The documentation link between the keys and the `attributes()` method could be stronger.
I would either refer to the `attributes()` method in the keys' documentation, or list the two canonical keys in the `attributes()`'s documentation.